web/templates/confirm_dialog/confirm_delete_detached_attachments.hbs

```diff
@@ -0,0 +1,11 @@
+<p>
+    {{t "Would you like to delete the following uploaded files? If you do so, they will no longer be accessible from the edit history for this message."}}
+    <a class="uploaded_files_settings_link" href="/#settings/uploaded-files">{{t "Manage uploaded files" }}</a>
+</p>
+<ul>
+    {{#each attachments_list}}
+        <li>
+            <a href="/user_uploads/{{{this.path_id}}}" target="_blank">{{{this.name}}}</a>
```

Comment:
This needs `rel="noopener noreferrer"` -- using `target=_blank` without that is a security bug.

Also, why are using `{{{` in the template here? Using it is often an XSS bug.